config/elvis.config

```diff
@@ -26,6 +26,11 @@
                     #{regex => "^([a-z][a-z0-9]*_?)*(_SUITE)?$",
                       ignore => []}
                   },
+                  {
+                    elvis_style,
+                    function_naming_convention,
+                    #{regex => "^([a-z][a-z0-9]*_?)*(_SUITE)?$"}
```

Comment:
Why is the `(_SUITE)` present for the `module_naming_convention`? I was trying to follow the pattern established from module names by `(_SUITE)` being present [in elvis.config](https://github.com/kbaird/elvis/blob/Issue%23186-FunctionNames-Refactoring/config/elvis.config#L26) but absent from [style_SUITE.erl](https://github.com/kbaird/elvis/blob/Issue%23186-FunctionNames-Refactoring/test/style_SUITE.erl#L290)

If `(_SUITE)` is removed from [the function use](https://github.com/kbaird/elvis/blob/Issue%23186-FunctionNames-Refactoring/config/elvis.config#L32), should it also be removed from [the module use](https://github.com/kbaird/elvis/blob/Issue%23186-FunctionNames-Refactoring/config/elvis.config#L26)?